src/include/daos_srv/container.h

```diff
@@ -95,6 +91,7 @@ struct ds_cont_child {
 	uint64_t		 sc_aggregation_max;
 	uint64_t		*sc_snapshots;
 	uint32_t		 sc_snapshots_nr;
+	uint32_t		 sc_open;
```

Comment:
dtx_batched_commit_register/dtx_batched_commit_deregister is per container based, not per cont-child based.